pkg/sanity/sanity.go

```diff
@@ -285,15 +285,10 @@ func (sc *TestContext) Setup() {
 
 	By("creating mount and staging directories")
 
-	// If callback function for creating target dir is specified, use it.
-	targetPath, err := createMountTargetLocation(sc.Config.TargetPath, sc.Config.CreateTargetPathCmd, sc.Config.CreateTargetDir, sc.Config.CreatePathCmdTimeout)
```

Comment:
I think we still need these because the staging directory is the responsibility of the CO, and the publish path parent directory is the responsibility of the CO

Reply:
Since we're no longer creating the staging path in NodeStageVolume, I've returned creation of this path to the sanity test; however, if we attempt to initialize the targetPath it will fail since this is completed during the NodePublishVolume calls...

Reply:
We discussed this offline - after adjusting `NodeUnpublishVolume` to remove any created directories the tests pass without modification. This section has been reverted.